readthedocs/starterguides/developerguides/object_lifecycle_management_developerguide.md

```diff
@@ -0,0 +1,81 @@
+# Developer Guide: Object Lifecycle Management
+# Overview
+This guide is intended for Object Lifecycle feature developers. It provides detailed steps to write new API using OpenSDS. To understand the feature and requirements in detail, please refer to the Object Lifecycle Management user guide and Object Lifecycle design doc. 
```

Comment:
It's not used to write new API using OpenSDS, but to do secondary development based on existed lifecycle APIs.